README.md

```diff
@@ -5,7 +5,7 @@
 [![NPM](https://nodei.co/npm/pouchdb-hoodie-sync.png)](https://npmjs.org/package/pouchdb-hoodie-sync)
 
 [![Build Status](https://travis-ci.org/hoodiehq/pouchdb-hoodie-sync.svg?branch=master)](https://travis-ci.org/hoodiehq/pouchdb-hoodie-sync)
-[![Coverage Status](https://coveralls.io/repos/hoodiehq/pouchdb-hoodie-sync/badge.svg?branch=master)](https://coveralls.io/r/hoodiehq/pouchdb-hoodie-sync?branch=master)
+[![Coverage Status](https://coveralls.io/github/hoodiehq/pouchdb-hoodie-sync?branch=master)](https://coveralls.io/github/hoodiehq/pouchdb-hoodie-sync?branch=master)
```

Comment:
Please don’t change the image URL (the first one): https://coveralls.io/repos/hoodiehq/pouchdb-hoodie-sync/badge.svg?branch=master, only the second one (as you did correctly :)